### ISSUE TITLE
Inquirer cannot be found

### ISSUE THREAD

**Issue by ghaiklor** [yeoman/generator#866]
When you moved `inquirer` to `devDependencies` you made a mistake. `inquirer` is using for user prompts. So my tests is failing with error:

> Cannot find module `inquirer`

**Comment by ghaiklor** [comment]
Summoning @sindresorhus who has made this change in 11255c7b50f6fc85ad74eb6047579e53320b124f

**Comment by kevva** [comment]
Yup, seems like it's needed in https://github.com/yeoman/generator/blob/f8a5fc9bc456685cf7a16c3139027ccc13da6355/lib/test/adapter.js#L4.

**Comment by ghaiklor** [comment]
@kevva and here - https://github.com/yeoman/generator/blob/f8a5fc9bc456685cf7a16c3139027ccc13da6355/lib/base.js#L225-L239

```javascript
Base.prototype.prompt = function (questions, callback) {
  questions = promptSuggestion.prefillQuestions(this._globalConfig, questions);

  this.env.adapter.prompt(questions, function (answers) {
    if (!this.options['skip-cache']) {
      promptSuggestion.storeAnswers(this._globalConfig, questions, answers);
    }

    if (_.isFunction(callback)) {
      callback(answers);
    }
  }.bind(this));

  return this;
};
```

**Comment by sindresorhus** [comment]
Sorry about that. I read `lib/test/adapter.js` as `test/adapter.js` when grepping for it. Though in all fairness the tests should have caught it. Fixed in 0.21.1.
